### ISSUE TITLE
Play ONCE only Command / play very other bar etc. /  Difinate Chance / Poly Rhythms

### ISSUE THREAD

**Issue by kenspocket** [polyend/TrackerIssues#1288]
Command to play the note or FX CC PC , only once per pattern or every other bar or 2 bars etc.

so a given lane note , plays only one time the first time it plays,  like using Chance , but will trigger only once.

elektron devices have part of this.  i.e trigger every other bar/s , within chance selection dial.

uses: 

1: ONCE ONLY
PC , Program change ,  Once only Trigger only once per pattern start / selection. 

The external Midi fx / synth unit will not keep reloading the patch every loop, rather will load once per event only. 

2: EVERY OTHER BAR / increments 
Crash Cymbol only every 8 bars.
short patterns have longer outcomes.

3:EVERY OTHER STEP / increments
Play a syncopated poly rhythm from one note entry . within a 8 step pattern. OTHER STEP 3/4 etc , without using phrases / 3step and chance command 10% to produce "botch" effect.

**Comment by thumperjack** [comment]
+1

**Comment by itzaks** [comment]
Yes. Chance by percentage is not very musical and I would also love to have conditional trigs like this. 

Combined with [https://github.com/polyend/TrackerBetaTesting/issues/1456](individual track lengths) and you could create a lot of variation and movement without having a very long pattern altogether.

**Comment by felart** [comment]
+1
